requirements.txt

```diff
@@ -6,7 +6,9 @@
 #
 autopep8==1.3.1
 click==6.7                # via pip-tools
-django==1.10.6
+configparser==3.5.0       # via flake8
+django==1.11
+enum34==1.1.6             # via flake8
```

Comment:
> Do you mean that these packages (configparser, enum34, mccabe, etc) get installed not because they are direct dependencies to your project, but rather because they are dependencies of flake8?

Correct.

It is only for development/linting purposes. Actually nothing in this requirements.txt file is need to run the project because each sub project takes care of its own environment within the docker containers. I think in the past some of these were needed and are no longer. I've made an issue to clean that up: https://github.com/data-refinery/data_refinery/issues/17